src/content/docs/fr/guides/integrations-guide/vue.mdx

```diff
@@ -194,3 +195,19 @@ export default defineConfig({
   ],
 });
 ```
+
+### devtools
+
+<p><Since pkg="@astrojs/vue" v="4.2.0" /></p>
+
+Vous pouvez activer [Vue DevTools](https://devtools-next.vuejs.org/) dans le développement en passant un objet avec `devtools : true` à votre configuration d'intégration `vue()` :
```

Comment:
`in` -> `dans`
`in` -> `en` :white_check_mark: (`en développement en passant` c'est redondant)
donc `durant le développement` me semble plus clair.
```suggestion
Vous pouvez activer [Vue DevTools](https://devtools-next.vuejs.org/) durant le développement en passant un objet avec `devtools : true` à votre configuration d'intégration `vue()` :
```